python_apps/pypo/pypo/pypofetch.py

```diff
@@ -357,7 +357,7 @@ def process_schedule(self, schedule_data):
 
 
             self.media_prepare_queue.put(copy.copy(media_filtered))
-        except Exception, e: self.logger.error("%s", e)
+        except Exception as e: self.logger.error("%s", e)
```

Comment:
For code style reasons I would make a 2-liner out of it.
Also, you can pass the exception to `logger.error()` as it will take the string representation automatically.

```suggestion
        except Exception as e:
            self.logger.error(e)
```